laminci/_db.py

```diff
@@ -27,8 +27,7 @@ def setup_local_test_postgres(name: str = "pgtest", version: Optional[str] = Non
         version = ""
     process = run(
         f"docker run --name {name} -e POSTGRES_PASSWORD=pwd"
-        f" -e POSTGRES_DB={name} -d -p 5432:5432 postgres{version}",  # noqa
-        shell=True,
```

Comment:
And this here broke lamindb CI on unit tests because the postgres docker couldn't be spun up anymore.

<img width="1342" alt="image" src="https://github.com/user-attachments/assets/d23402d0-f7c4-4581-81d2-98ee8670b213" />

Reply:
Fixed: https://github.com/laminlabs/laminci/commit/4ebcede252192ea8bd1b3bf02f09622315f8d08d

Reply:
Wow, sorry!

Reply:
No worries!